src/components/bs5/linkColumns/linkColumns.stories.js

```diff
@@ -0,0 +1,67 @@
+// ComponentExample.stories.js
+/**
+ * @file linkColumns.stories.js
+ * @description Storybook configuration file for the link Columns component.
+ * @module linkColumns.stories
+ */
+
+import { LinkColumns } from "./linkColumns.js";
+import defaultdata from "./linkColumns.data.json";
+
+export default {
+  tags: ["autodocs"],
+  title: "3. Components/Link Columns",
```

Comment:
Can you align to vanilla naming.  "3. Components/Link List/Link List (Columns)"

as there is also Link List component. https://www.designsystem.qld.gov.au/styles/typography#section__link-lists

Also add link to https://www.designsystem.qld.gov.au/components/link-columns-link-list

https://github.com/qld-gov-au/qgds-vanilla/blob/main/src/components/_global/css/link_columns/stories/linkColumns.stories.js#L19C23-L19C52

https://qld-gov-au.github.io/qgds-vanilla/storybook/?path=/docs/2-globals-link-list-link-list-columns--overview